docs/how-to-guides/integrating-autoware/tuning-parameters-and-performance/important-temporal-performance-metrics.md

```diff
@@ -79,27 +79,27 @@ The following list shows the temporal performance metrics for Perception.
+| APER-014 | **Message rate** from CPP #10 to CPP #13             | Update rate of data sent from Clustering.                                                                 | Object Recognition        | 10 Hz           | Sensor Fusion relies on the data to be updated at expected frequency for data synchronization.                         |                                                                   |
+| APER-015 | **Message rate** from CPP #5 to CPP #13              | Update rate of data sent from Camera-based Object detection.                                              | Object Recognition        | 10 Hz           | Sensor Fusion relies on the data to be updated at expected frequency for data synchronization.                         |                                                                   |
+| APER-016 | **Response time** from CPP #6 to CPP #13             | Response time to consume data sent from Clustering after LiDARs output pointcloud.                               | Object Recognition        | N/A             | Sensor Fusion relies on fresh and up-to-date data for data synchronization.                            |                                                                   |
+| APER-017 | **Response time** from CPP #3 to CPP #13             | Response time to consume data sent from Camera-based Object detection after Cameras output images.               | Object Recognition        | N/A             | Sensor Fusion relies on fresh and up-to-date data for data synchronization.                            |                                                                   |
+| APER-018 | **Message rate** from CPP #10 to CPP #17             | Update rate of data sent from Clustering.                                                                 | Object Recognition        | 10 Hz           | Validator relies on the data to be updated at expected frequency for data synchronization.                             | It seems similar to APER-014, but the topic message is different. |
+| APER-019 | **Message rate** from CPP #12 to CPP #17             | Update rate of data sent from DNN-based Object Recognition.                                               | Object Recognition        | 10 Hz           | Validator relies on the data to be updated at expected frequency for data synchronization.                             |
+| APER-020 | **Response time** from CPP #6 to CPP #17 via CPP #10 | Response time to consume data sent from Clustering after LiDARs output pointcloud.                               | Object Recognition        | N/A             | Validator relies on fresh and update-date data for data synchronization.                               | It seems similar to APER-015, but the topic message is different. |
```

Comment:
It's not a comment for the line but for the diagram.
In the diagram, "Validation" should be "Validator"